scripts/jenkins/update_automation

```diff
@@ -10,28 +10,41 @@
 # Usage: update_automation [<scriptname>, ...]
 #
 # J. Daniel Schmidt <jdsn@suse.de>
+# Bernhard M. Wiedemann <bwiedemann suse.de>
 #
 
-automation_repo="git://github.com/SUSE-Cloud/automation.git"
-
-rpm -q git-core || zypper --non-interactive in git-core
+: ${automation_repo:="git://github.com/SUSE-Cloud/automation.git"}
+: ${branch:=master}
 
 pushd ~ > /dev/null
 a_dir=${automation_repo#*git://}
 a_dir=${a_dir%.git}
 p_dir=${a_dir%/*}
 
+function do_wget
+{
+    wget -Omaster.tar.gz https://${a_dir}/archive/${branch}.tar.gz
+    tar xzf master.tar.gz
+    mkdir -p $p_dir
+    rm -rf $a_dir
+    mv automation-$branch $a_dir
+    exit 0
```

Comment:
I guess that the goal is simply end the script at this point. This makes me wonder why not to use only the github file from master as a single solution. Now there is not advantage AFACS for `git`

In any case I will suggest to `exit` outside the function, this breaks the expectations.